orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/model/PluginArtifact.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.front50.model;
+
+import java.util.List;
+import java.util.regex.Pattern;
+import javax.annotation.Nonnull;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class PluginArtifact {
+  @Nonnull private String id;
+  private String description;
+  private String provider;
+  @Nonnull private List<Release> releases;
+  private Long createTs;
+  private Long lastModified;
+  private String lastModifiedBy;
+
+  @Data
+  public static class Release {
+    public static final Pattern SUPPORTS_PATTERN =
+        Pattern.compile(
+            "^(?<service>[\\w\\-]+)(?<operator>[><=]{1,2})(?<version>[0-9]+\\.[0-9]+\\.[0-9]+)$");
+    public static final String SUPPORTS_PATTERN_SERVICE_GROUP = "service";
+    public static final String SUPPORTS_PATTERN_OPERATOR_GROUP = "operator";
+    public static final String SUPPORTS_PATTERN_VERSION_GROUP = "version";
```

Comment:
I don't think we need this. Orca shouldn't be doing validation of models that are owned by other services, so it won't need to do any of this parsing.

Reply:
Oh yea, I think I probably deleted this right as you left that message - it's gone now.